brunch-config.js

```diff
@@ -31,8 +31,7 @@ exports.config = {
   plugins: {
     babel: {
       // Do not use ES6 compiler in vendor code
-      ignore: [/^(web\/static\/vendor)/],
-      loose: "all"
```

Comment:
:( Is all this really necessary. Do we need the loose option at all? This list seems like a guaranteed churn area for future updates